python/exercise1.1,py

```diff
@@ -0,0 +1,3 @@
+familyname = input("What is your family name:")
```

Comment:
should file name be `exercise1-1.py` rather than `exercise1.1,py`?
- We should use `.py` rather than `,py` as extension file name.
- We usually don't use `.` in file name except before the files extension name.